app/controllers/enrollments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class EnrollmentsController < ApplicationController
+  before_action :authenticate_player!
+
+  def create
+    current_player.join_game(@current_game)
+  end
+
+  private
+
+  def current_game
+    @current_game ||= Game.is_available.find(params[:game_id])
```

Comment:
Just curious -- is the is_available necessary here? Or is it just for safety? (so we're not accidentally getting any games that are taken)